gapic/batch/common.yaml

```diff
@@ -1,5 +1,15 @@
 common:
   batch_apis: '*'
-  batch_languages: '*'
   api_config_pattern: ${GOOGLEAPIS}/gapic/api/artman_${API_SHORT_NAME}.yaml
   artman_language_yaml: ${GOOGLEAPIS}/gapic/lang/common.yaml
+  git_repos:
```

Comment:
This is because artman currently requires the `git_repos` arg - but it is better handled in artman itself, rather than with empty configuration. Removed.